debug_toolbar/static/debug_toolbar/js/toolbar.timer.js

```diff
@@ -20,7 +20,7 @@ function addRow(stat, endStat) {
             "<td>" +
             stat.replace("Start", "") +
             "</td>" +
-            '<td class="djdt-timeline"><svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none"><rect y="0" height="5" fill="#ccc" /></svg></td>' +
+            '<td><svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none"><rect y="0" height="5" fill="#ccc" /></svg></td>' +
```

Comment:
I'm confused. Do these `td`'s not require a `djdt-width-30` CSS class now?

Reply:
Correct. The styling was already handled by the `<colgroup>` element which applies to the entire column:

![timer](https://user-images.githubusercontent.com/347634/96201658-02754580-0f12-11eb-95e4-9a2ef14298b6.png)